recipe/meta.yaml

```
@@ -33,8 +33,8 @@ requirements:
     - ml_dtypes
   run_constrained:
     - tensorflow >=2.15.0,<2.17.0a
-    - jax 0.4
-    - torch 2.1
+    - jax 0.4.*
+    - torch 2.1.*
```

By the way, while are at it, we may want to fix this name. `torch` is not a conda-forge package name so this constraint is not doing anything, the correct name here is `pytorch`, see https://github.com/conda-forge/pytorch-cpu-feedstock . fyi @conda-forge/keras 
```suggestion
    - pytorch 2.1.*
```